bootloader.mk

```diff
@@ -125,6 +125,13 @@ ifeq ($(strip $(BOOTLOADER)), apm32-dfu)
     DFU_ARGS ?= -d 314B:0106 -a 0 -s 0x08000000:leave
     DFU_SUFFIX_ARGS ?= -v 314B -p 0106
 endif
+ifeq ($(strip $(BOOTLOADER)), wb32-dfu)
+    OPT_DEFS += -DBOOTLOADER_WB32_DFU
+
+    # Options to pass to dfu-util when flashing
+    DFU_ARGS ?= -d 342D:DFA0 -a 0 -s 0x08000000:leave
```

Comment:
Does this actually support dfu-util?

Reply:
dfu-util is not supported, can I delete this code directly ?

Reply:
The dfu-util related sutff can be deleted but it would be useful to keep the `wb32-dfu` bootloader option.